install-and-configure/advanced-configuration/key-rotation.md

```diff
@@ -47,7 +47,7 @@ Thanos federation makes use of the `kubecost-thanos` Kubernetes secret as descri
 
 ## ETL backup keys
 
-ETL backups rely on the secret defined by the Helm value `.Values.kubecostModel.etlBucketConfigSecret`. More details can be found on the [ETL backup page](/install-and-configure/install/etl-backup/etl-backup.md).
+ETL backups rely on the secret defined by the Helm value `.Values.kubecostModel.etlBucketConfigSecret`. More details can be found in the [Federated ETL Backups and Alerting](/install-and-configure/install/etl-backup/etl-backup.md) doc.
```

Comment:
Need someone to verify if this is still accurate in v2.0. `Values.kubecostModel.etlBucketConfigSecret` still exists in the helm chart but just want to confirm.

Reply:
We don't mention this flag in the article we link to, so some of this content might need to be rewritten.